support/scheduler-client/client.go

```diff
@@ -0,0 +1,89 @@
+//
+// Copyright (c) 2017 Tencent
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package scheduler
+
+import (
+	"bytes"
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"github.com/edgexfoundry/edgex-go/support/logging-client"
+	"io"
+	"net/http"
+)
+
+const (
+	SchedulerClientName = "support-scheduler-client"
+)
+
+// Common http const
+const (
+	HttpPostMethod     = "POST"
+	ContentType        = "Content-Type"
+	ContentTypeJsonVal = "application/json"
+)
+
+var loggingClient logger.LoggingClient = logger.NewClient(SchedulerClientName, false, "")
```

Comment:
Libraries should try to not write logs directly. In this case you are only logging before returning an error, so the user of this client will have the same information to create the log.
SchedulerClientName should not be needed.

Reply:
@feclare  you mean I don't need to use `LoggingClient` in scheduler's client?

Reply:
Yes, it is preferable to not log directly from the libs if not really needed. Like in this case.